src/main/java/org/truffleruby/core/module/ModuleNodes.java

```diff
@@ -2093,33 +2093,33 @@ protected void setMethodVisibility(DynamicObject module, Object name,
         @Child private CallBlockNode callBlockNode = CallBlockNode.create();
 
         @Specialization
-        protected DynamicObject refine(DynamicObject self, DynamicObject classToRefine, NotProvided block) {
+        protected DynamicObject refine(DynamicObject self, DynamicObject objectToRefine, NotProvided block) {
```

Comment:
Not sure about the name `objectToRefine`. We should rename `classToRefine` but maybe just to `object`

Reply:
How about `moduleToRefine`? Classes are Modules (also holds for isRubyClass() implies isRubyModule()).

Reply:
The terminology is documented in https://github.com/oracle/truffleruby/blob/e276e66f38306842602833fb2357a9416580c889/src/main/java/org/truffleruby/core/module/ModuleFields.java#L80-L93

It's for an example like:
```ruby
module M # M is the namespace module
  R = refine C do # C is the refined module
  end
  # R is the refinement module
end
```

So I think `refinedModule` would be the best, and the `refinedClass` field in ModuleFields should be renamed to `refinedModule`.